### PR TITLE
refactor(SettingsAccounts): code

### LINKED ISSUE #388: settings-accounts

### PR DESCRIPTION
**Pull request by samandarkuchkarov**
Fixes #388 

## Proposed Changes

-Change the imports to absolute:
-Move text to i18n
-Change the colors from the constants to similar ones in Color
-Transferring the screen logic to components
-Replace navigation hooks with typed navigation hooks in folder src/hooks

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

File: src/components/settings-accounts.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    return (
+      <View style={styles.emptyContainer}>
+        <NoTransactionsIcon
+          color={getColor(Color.graphicSecond3)}
+          style={styles.space}
+        />
+        <Text
+          t14
+          i18n={I18N.settingsAccountNoWallet}
+          color={getColor(Color.textSecond1)}
```
Comment (gHashTag) [inline src/components/settings-accounts.tsx:28]: `сolor={Color.textSecond1}`

File: src/components/wallet-row.tsx [hunk 2]
```diff
@@ -1,44 +1,50 @@
...
+const CARD_WIDTH = 78;
+const CARD_RADIUS = 8;
 
 export const WalletRow = ({item, onPress}: WalletRowProps) => {
   const style = useMemo(
     () => (item.isHidden ? {opacity: 0.5} : {}),
     [item.isHidden],
   );
 
+  const pressCard = () => {
```
Comment (gHashTag) [inline src/components/wallet-row.tsx:24]: Can be written in 1 line

File: src/screens/settings-accounts.tsx [hunk 3]
```diff
@@ -1,19 +1,11 @@
...
-
+import {SettingsAccounts} from '@app/components/settings-accounts';
 import {useWallets} from '@app/hooks';
-import {LIGHT_GRAPHIC_SECOND_3, LIGHT_TEXT_SECOND_1} from '@app/variables';
-
-import {NoTransactionsIcon, Text} from '../components/ui';
-import {WalletRow} from '../components/wallet-row';
-import {Wallet} from '../models/wallet';
-import {RootStackParamList} from '../types';
+import {useTypedNavigation} from '@app/hooks';
```
Comment (gHashTag) [inline src/screens/settings-accounts.tsx:5]: use one import for hooks

File: src/components/wallet-row.tsx (hunk 2 above)
Comment (samandarkuchkarov) [reply src/components/wallet-row.tsx]: done

File: src/components/settings-accounts.tsx (hunk 1 above)
Comment (samandarkuchkarov) [reply src/components/settings-accounts.tsx]: done

File: src/screens/settings-accounts.tsx (hunk 3 above)
Comment (samandarkuchkarov) [reply src/screens/settings-accounts.tsx:5]: done

**Review by gHashTag — CHANGES_REQUESTED** [review]
Fix comment

File: src/screens/settings-accounts.tsx [hunk 4]
```diff
@@ -1,19 +1,11 @@
 import React, {useCallback, useEffect, useState} from 'react';
 
-import {useNavigation} from '@react-navigation/native';
-import {StackNavigationProp} from '@react-navigation/stack';
-import {FlatList, StyleSheet, View} from 'react-native';
-
+import {SettingsAccounts} from '@app/components/settings-accounts';
 import {useWallets} from '@app/hooks';
```
[the remark this answers is not in the document]
Comment (samandarkuchkarov) [reply src/screens/settings-accounts.tsx]: done

**Pull request merged**